in_text/text_02_drivers.yml

```diff
@@ -107,7 +107,7 @@ entities:
       data-max: 100
       data-units: NA
     -
-      attr-label: Windspeed
+      attr-label: WindSpeed
       attr-def: >-
         Speed of the wind above the reservoir, calculated to a daily number using the methods described in Winslow and others 2017.
```

Comment:
I don't see a reference for Winslow and others 2017

Reply:
Good Catch. I pulled the citation from last year's data release and added it to `text_02_drivers.yml` and found in https://github.com/USGS-R/forecast-data-release-v2/pull/13/commits/b06cc0d16c2457a821343d69b21908958e2b55ff

Reply:
I believe the linked citation is the correct source but @limnoliver  should confirm